### DISCUSSION TITLE
Revalidate problem

### DISCUSSION THREAD

**Discussion by azkar-sh** [community/community#138250] [GitHub Education]
### Hi everyone,

Hi Everyone
I still eligible to enroll the github Education for Student, the problem is when I take a picture to prove my student status, I got rejected because the photo taken is not clear and I just have document only which is too small and not that clear too be seen from camera. Is there any support email address so I can send them immediately?

**Comment by alx-mp4** [comment]
Hi @azkar-sh, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
